src/nuclio/projects/project/functions/version/version-configuration/tabs/version-configuration-build/version-configuration-build.tpl.html

```diff
@@ -42,7 +48,7 @@
                 </igz-validating-input-field>
             </div>
             <div class="igz-col-50 build-field build-onbuild-image-field">
-                <div class="field-label">Onbuild image</div>
+                <div class="field-label label-with-tooltip align-items-center">Onbuild image</div>
```

Comment:
you added `label-with-tooltip` class, but where is tooltip?

Reply:
This is necessary for alignment

Reply:
But there isn't any tooltip, maybe `align-items-center` will be enough for alignment?